extensions/ql-vscode/src/remote-queries/remote-queries-markdown-generation.ts

```diff
@@ -152,14 +160,42 @@ function buildMarkdownAlertMessage(interpretedResult: AnalysisAlert): string {
         token.location.fileLink,
         token.location.highlightedRegion?.startLine,
         token.location.highlightedRegion?.endLine,
-        token.text,
+        token.text
       );
     }
   }
   // Italicize the alert message
   return `*${alertMessage}*`;
 }
 
+function generateMarkdownForPathResults(
+  interpretedResult: AnalysisAlert,
+  language: string
+): MarkdownFile {
+  const pathLines: MarkdownFile = [];
+  for (const codeFlow of interpretedResult.codeFlows) {
+    const stepCount = codeFlow.threadFlows.length;
+    pathLines.push(`#### Path with ${stepCount} steps`);
+    for (let i = 0; i < codeFlow.threadFlows.length; i++) {
```

Comment:
Feel free to ignore this but you could use the `stepCount` here
```suggestion
    for (let i = 0; i < stepCount; i++) {
```